docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -3526,7 +3526,7 @@ EMPTY_CLOB()
 
 * If EMPTY_CLOB() is specified, then a dummy lob handle is created.
 
-** No data is associated with the empty LOBs yet, but these dummy LOB handles can later be used to populate with new LOB data. If the LOB had data previously associated with it, it will be erased.
+** No data is associated with the empty LOBs yet, but these dummy LOB handles can later be used to populate the LOB with new data. If the LOB had data previously associated with it, it will be erased.
 
 ** The dummy LOB handle will get the same datatype as the underlying column.
 +
```

Comment:
Here, maybe you can elaborate by  explaining what  "getting the same datatype as column " means .  If the column is declared as type LOB 'external' in the create statement, the dummy lob handle will get the type external and this LOB can only handle LOB's that are external. 
If the column is declared without the external clause, the LOB data will be maintained within the Trafodion space in the internal location maintained by Trafodion.

Reply:
Thanks Sandhya, I've updated this paragrahy based on you suggestions. :)